backend/README.md

```diff
@@ -15,7 +15,9 @@ and the web accessible Swagger API documentation.
 ### Requirements
 
 Project dependencies are listed in the `requirements.txt` file at the project's
-root structure.
+backend root structure.
+
+you do not need to run this command if you have already ran the ./setup.sh script in the Rosalution root directory.
```

Comment:
This and the example below can be revised to discuss how to activate the python venv to run the python tools.

Reply:
I think I have made the requested change.

Reply:
I think if someone wanted to install the python virtual environment without using the setup.sh script, those commands should be outlined in the readme where the original `pip install -r requirements.txt` is.

Reply:
After trying out the virtual environment for python, perhaps a more detailed explanation how the virtual environment works as Ange is pointing out. 

For example, the setup.sh script creates a virtual environment for a single terminal session. Once it closes, then the terminal no longer has knowledge of the virtual environment and needs to be re-sourced each time you work with the python tools in the backend.

Reply:
We can probably circle back on how we want to document and tidy things up.  This works for getting things started for now and we can move on to some other repo cleanup work.